cmd/controller/run.go

```diff
@@ -266,3 +276,21 @@ func Run(opts Options, runLog logr.Logger) error {
 
 	return nil
 }
+
+// parseTlsCipherSuites tries to validate and return the user-input ciphers or returns a default list
+// defaults stolen from: https://github.com/vmware-tanzu/community-edition/blob/main/addons/packages/antrea/1.5.3/bundle/config/values.yaml#L48
```

Comment:
I have a slight maintenance concern about hardcoding the defaults. Primarily, how will we remember to update them?

A couple of mitigations/options that come to mind:
1. Keep the hardcoding
    - add a "release step" as a reminder to check these
    - or some other reminder (e.g. calendar/slack reminder, "when we bump go" reminder, etc.)
2. Remove the hardcoding and try to pull a "safe" list from somewhere. The best option that comes to mind for me is the CipherSuites function (https://go.dev/src/crypto/tls/cipher_suites.go). I think this would help keep the ciphers updated as we bump our deps.

This is a non-blocking comment. I'll leave it to y'all to figure out what you're comfortable with.

Reply:
cool - i had a related conversation with DK in slack where he recommended keeping golang's defaults via `""` . 

one thing this list allowed me to do was to verify that it's working as desired without doing all the plumbing yet (by deploying the code, and then at the terminal trying openssl with an excluded cipher and seeing a handshake failure).   


I'm happy to leave this list behind as the 'prototype' phase ends